### PR TITLE
Fixed the height of button in navbar

### LINKED ISSUE #3136: fix the height of button in navbar
### Environment

ToolJet Cloud

### What is the expected behaviour?

The heights of the buttons in the navbar should be equal.

### What is the current behaviour?

The `Workspace` button height is lower than the `User` button

### How to reproduce the issue?

1. Go to the dashboard
2. On the navbar - hower on the buttons to see the difference in height

### Screenshots or Screencast
Screenshot 2022-05-27 at 1 47 24 AM

### PR DESCRIPTION
**Pull request by Syed-Ansar**
Fixed the height of button in Navbar 

fixes #3136 

# Light Mode

Screenshot 2022-06-25 at 4 15 31 PM

# Dark Mode

Screenshot 2022-06-25 at 4 15 26 PM

### CODE REVIEW

**Comment by withshubh** [comment]
@Syed-Ansar - Thanks for contributing. 🎉 
Your PR will be reviewed soon.

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: frontend/src/_styles/theme.scss [hunk 1]
```diff
@@ -4852,6 +4852,11 @@ div#driver-page-overlay {
   }
 }
 
+.organization-list a{
```
[the remark this answers is not in the document]
Comment (Syed-Ansar) [reply frontend/src/_styles/theme.scss]: Yes added @gsmithun4 . I haven't noticed as it is scss. Thank you.

**Review by gsmithun4 — CHANGES_REQUESTED** [review]
Screenshot 2022-07-11 at 4 41 00 PM
Horizontal line is broken

**Comment by withshubh** [comment]
Hey @Syed-Ansar 
Can you please update the PR with the requested changes?

**Comment by withshubh** [comment]
Hey @Syed-Ansar 
Please update the PR with the suggested changes.

**Comment by withshubh** [comment]
Closing the PR as there was no update for more than a month.

**Pull request closed without merging**